core/src/main/kotlin/com/justai/jaicf/context/ActionContext.kt

```diff
@@ -73,4 +75,11 @@ open class ActionContext(
      * @param texts a list of String to select random from
      */
     fun Reactions.sayRandom(texts: List<String>) = say(random(texts))
+
+    infix fun ButtonsReaction?.toStates(states: List<String>) {
```

Comment:
Not sure about this extension, but we need some transition management for channel-specific buttons.